src/config_docker/encryption.ts

```diff
@@ -0,0 +1,6 @@
+// Encryption algorithm
+export const algorithm = "aes-256-ctr";
+
+// Encryption secret (strictly 32 length)
+export const secretKey =
+  process.env["SECRET_KEY"] || "h/f#Ts8w5sch5L:J*_gPW)]$'!4K.K.-";
```

Comment:
Please do not include a default secret key, use something longer than 32 to throw an error to bot hoster. To keep the data safer.